static/js/components/PersonalTab.js

```diff
@@ -104,7 +104,10 @@ export default class PersonalTab extends React.Component {
     const { profile, updateProfile } = this.props;
     let editState = _.cloneDeep(profile);
     let { payload: { profile: { image } }} = fetchResponse;
-    let newEditState = Object.assign({}, editState, { image });
+    let newEditState = {
+      ...editState,
+      image
+    };
```

Comment:
why not leave this as one line? I'm fine either way, I have a slight preference for leaving small object literals on one line though.